osdk/tools/docker/README.md

```diff
@@ -0,0 +1,38 @@
+# OSDK Development Docker Images
+
+OSDK development Docker images are provided to facilitate developing and testing OSDK project. These images can be found in the [asterinas/osdk](https://hub.docker.com/r/asterinas/osdk/) repository on DockerHub.
+
+## Building Docker Images
+
+To build a Docker image for OSDK and test it on your local machine, navigate to the root directory of the Asterinas source code tree and execute the following command:
+
+```bash
+cd <asterinas dir>
+# Build Docker image
+docker buildx build \
+    -f osdk/tools/docker/Dockerfile \
+    --build-arg ASTER_RUST_VERSION=$(grep "channel" rust-toolchain.toml | awk -F '"' '{print $2}') \
+    -t asterinas/osdk:$(cat VERSION)-$(date +%Y%m%d) \
```

Comment:
The Docker version is stored in the `DOCKER_IMAGE_VERSION` file. So the command should be `$(cat DOCKER_IMAGE_VERSION)`.